apps/fluent-tester/src/TestComponents/TextExperimental/V2Usage.ios.tsx

```diff
@@ -44,6 +48,7 @@ export const V2Usage: React.FunctionComponent = () => {
         <TextV1 variant="title1">Title1</TextV1>
         <LargeTitle>LargeTitle</LargeTitle>
         <TextV1 variant="largeTitle">LargeTitle</TextV1>
+        <LimitedDisplay>Disp(&le;70)</LimitedDisplay>
```

Comment:
What's with `Disp(&le;70)` ?

Reply:
This is getting replaced with a more obvious usage of `maximumFontSize`.